src/mixins/filing-template-mixin.ts

```diff
@@ -350,6 +352,13 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       }
     }
 
+    if (this.hasNatureOfBusinessChanged) {
+      filing.changeOfRegistration.business.naics = {
+        naicsCode: this.getCurrentNaics.naicsCode,
+        naicsDescription: this.getCurrentNaics.naicsDescription
+      }
+    }
```

Comment:
`filing.changeOfRegistration.business.naics = this.getCurrentNaics` ?

Reply:
Oh your right :) thank you

Reply:
Updated